mlrun/model_monitoring/db/_schedules.py

```diff
@@ -208,15 +208,19 @@ def get_endpoint_last_request(self, endpoint_uid: str) -> Optional[int]:
             return None
```

Comment:
See `get_endpoint_last_request`, I think that after your changes it returns float not int right? because `self._schedules` is not enriched with float timestamp. Please update the docstring accordingly